src/circuits/circuits-util.hpp

```diff
@@ -15,7 +15,7 @@ libsnark::linear_combination<FieldT> packed_addition(
     libsnark::pb_variable_array<FieldT> input);
 template<typename FieldT>
 libsnark::pb_variable_array<FieldT> from_bits(
-    std::vector<bool> bits, libsnark::pb_variable<FieldT> &ZERO);
+    std::vector<bool> bits, libsnark::pb_variable<FieldT> ZERO);
```

Comment:
Why do you pass by value rather than by reference?

Reply:
No real reason, will fix that.

Reply:
@AntoineRondelet There was issues with prftest due to it.

Reply:
Do you remember the issue?

Reply:
Just fixed it. Ther error was due to the fact that sometimes ZERO is not a const, unreferencing it made the trick now it is changed so that all ZERO are const.